content/questions/admin/question-010.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "What are the benefits and risks of using apps and actions from the GitHub Marketplace?"
```

Comment:
You probably meant to ask a different question here

Reply:
No, I took it from the exam preparation study guide.

Reply:
The question asks about GitHub Marketplace apps and actions and the answers are about SAML SSO..

Reply:
You put the same question in Q9

Reply:
It was copy pasted without changing the question.

Reply:
It was copy-pasted without changing the question. It should be ok now.